README.md

```diff
@@ -10,70 +10,71 @@ The Swiftest way to GraphQL
 - [Contributing](#contributing)
 - [License](#license)
 
-## Features 
+## Features
+
 - [ ] ⚡️ [Code Generation](#code-generation)
-- [x] 💒 [Database Agnostic](#database-agnostic)
-  - [x] 👑 [Realm](#realm)
-    - [x] 🦃 Realm Obj-C
-    - [ ] 🕊 Realm Swift
-  - [ ] 💾 [Core Data](#core-data)
 - [x] 🔨 [Query Builder](#query-builder)
 - [x] ⛑ [Type safe Mapping](#crust-for-type-safe-mapping)
 - [x] 🆒 [Type safe JSON](#jsonvalue-for-type-safe-json)
 - [x] 🐙 [Threading](#threading)
 - [x] 🌐 [Network Library](#network-library)
 - [ ] 🔮 Network Library Agnostic
 - [ ] 🥓 [Batched Queries](#batched-queries)
-- [x] ❓ Supports Optional Types and Collections.
 
-AutoGraph is a Swift client framework for making requests using GraphQL and mapping the responses to strongly typed models. Models may be represented by any type, including database model objects such as Realm or Core Data models. AutoGraph relies heavily on Swift's type safety to drive it, leading to safer, compile time checked code.
+AutoGraph is a Swift client framework for making requests using GraphQL and mapping the responses to strongly typed models. Models may be represented by any `Decodable` type. AutoGraph relies heavily on Swift's type safety to drive it, leading to safer, compile time checked code.
 
 ## Requirements
+
 Swift 5.0
-* Swift 4.2+ - use version `0.6.0`+.
-* Swift 4.1.2 - use version `0.5.1`.
-* Swift 4.0 - use version `0.4.6`.
-* Swift 3 use version `0.3.0`.
+
+- Swift 5.0 pre Decodable - use version `0.8.0`
```

Comment:
FYI, i'm going to release an 0.8.0 in the old way of mapping just to cover swift 5 there. dev-0.8 branch will end up actually releasing to 0.9.0